xcode/Subconscious/Shared/Components/Common/SlashlinkDisplayView.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  SlashlinkDisplayView.swift
+//  Subconscious
+//
+//  Created by Gordon Brander on 5/5/23.
+//
+
+import SwiftUI
+
+/// A view for slashlinks, optimized for compact display
+struct SlashlinkDisplayView: View {
+    var slashlink: Slashlink
+    var baseColor = Color.accentColor
+    var slugColor = Color.accentColor
+    var labelColor = Color.secondary
+    
+    var body: some View {
+        HStack(spacing: 0) {
```

Comment:
It's setting a max line limit on all children.

In general, though, I have a tendency to make sure all views return a single view at the top level, since I had problems with switch statements that return heterogeneous views in earlier versions of SwiftUI (I think this has since been resolved, so this is probably superstition on my part).